src/Microsoft.Identity.Abstractions/DownstreamApi/IDownstreamApi.cs

```diff
@@ -236,5 +237,136 @@ public partial interface IDownstreamApi
             Action<DownstreamApiOptions>? downstreamApiOptionsOverride = null,
             CancellationToken cancellationToken = default)
             where TOutput : class;
+
+#if NET8_0_OR_GREATER
```

Comment:
Since you are always referencing System.Text.Json on all TFMs, you don't need to only put these APIs in for `net8`. You can remove the `#if` and always compile them in.

That also means you don't need an explicit target for `net8` since all targets will have these new methods.

Reply:
Switched back to net8+ only to avoid breaking existing users